### ISSUE TITLE
Plans for sounds/audio feedback?

### ISSUE THREAD

**Issue by GraionDilach** [Mailaender/OpenHV#1]
I'm asking out of interest. You have made a clear cut role on the artwork side but I can't see anything on the sounds side.

Generally, feel free pinging me if you need help, it would be cool to see something original being built upon the engine.

**Comment by Mailaender** [comment]
My plan would be that it has to be freely licensed as in the [Open Source definition](https://opensource.org/osd) but I have no idea what would be fitting. Maybe the [Tyrian sound effects](https://www.sounds-resource.com/pc_computer/tyrian/sound/6422/)?

**Comment by GraionDilach** [comment]
I wouldn't use anything directly from that site, that rips the games for assets so it might not be compatible with the project's OS definition. Could be used as a reference for the styles though.

I've got a few thumping sounds which might pass as explosions which I made half a decade ago by shutting my previous windows. I've also made a few tongue SFX which might be passed as placeholders until anything better. If you have an explicit license suggestion, I can dig those stuff up and then donate them in a PR along with placeholder voiceovers I could record (basically just a "Yes? What? Okay. Alright." set maybe in two accents - I got quality gear for this, since I voiced in C&C mods before).

I'll also probably look around for CC0 stuff - that worked back then during https://github.com/OpenRA/OpenRA/pull/8842.
